### PR TITLE
Added proxy for cip-email

### PR DESCRIPTION
**Pull request by mar36tin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+- cip-email-verification
+- cip-email-history
+- cip-email-insights
+
+### Testing
+#### Unit tests
+`sbt clean test`
+
+#### Integration tests
+`sbt clean it:test`
```
Comment (johno1985) [inline README.md:17]: I believe this should be sbt IntegrationTest/test

File: app/uk/gov/hmrc/cipemail/models/Email.scala [hunk 2]
```diff
@@ -14,18 +14,15 @@
...
 
-@Singleton()
-class MicroserviceHelloWorldController @Inject()(cc: ControllerComponents)
-    extends BackendController(cc) {
+case class EmailAddress(email: String)
 
-  def hello(): Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(Ok("Hello world"))
-  }
+object EmailAddress {
```
Comment (johno1985) [inline app/uk/gov/hmrc/cipemail/models/Email.scala:23]: did you say this was only needed in the unit test? if so, can it be moved into the unit test?

**Review by johno1985 — COMMENTED** [review]
comments added

File: README.md [hunk 3]
```diff
@@ -1,8 +1,34 @@
...
+
+### Running app
+
+In order to run this microservice cip-phone-number you willl need to run the downstream services first. Then run
```
[the remark this answers is not in the document]
Comment (johno1985) [reply README.md:21]: may also be worth adding an instruction to use service manager to start up the required services

File: README.md (hunk 1 above)
Comment (mar36tin) [reply README.md:17]: They both work!

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (johno1985) [reply README.md:21]: still a typo here

File: app/uk/gov/hmrc/cipemail/models/Email.scala [hunk 4]
```diff
@@ -14,18 +14,15 @@
...
+  implicit val format = Json.format[EmailAddress]
+
+  val phoneNumberReads: Reads[EmailAddress] =
+    (__ \ "phoneNumber").read[String].map(EmailAddress.apply)
```
Comment (johno1985) [inline app/uk/gov/hmrc/cipemail/models/Email.scala]: can this be moved into the unit test?

File: it/uk/gov/hmrc/cipemail/ValidateControllerIntegrationSpec.scala [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+//    "return 400 with invalid email address" in {
+//      val result = controller.validateEmailNumber()(
+//        fakeRequest.withBody(Json.toJson(EmailAddress("test.com"))))
+//
+//      status(result) shouldBe Status.BAD_REQUEST
+//      (contentAsJson(result) \ "message" ).as[String] shouldBe "Enter a valid email address"
+//    }
+
+  }
+}
```
Comment (johno1985) [inline it/uk/gov/hmrc/cipemail/ValidateControllerIntegrationSpec.scala:59]: no new line

**Review by brdgeth — DISMISSED** [review]

File: README.md [hunk 6]
```diff
@@ -1,8 +1,34 @@
...
+### Testing
+#### Unit tests
+`sbt clean test`
+
+#### Integration tests
+`sbt clean it:test`
+
+### Running app
+
+In order to run this microservice cip-email you willl need to run the downstream services first. Then run
```
Comment (johno1985) [inline README.md:21]: typo

**Review by johno1985 — COMMENTED** [review]
comment added + unresolved comments

**Review by brdgeth — APPROVED** [review]

**Pull request merged**